app/assets/stylesheets/govuk_publishing_components/components/_image-card.scss

```diff
@@ -180,3 +201,11 @@
     @include govuk-font(19);
   }
 }
+
+.gem-c-image-card--no-image {
+  .gem-c-image-card__text-wrapper {
+    @include govuk-media-query($from: mobile, $until: tablet) {
+      float: left;
```

Comment:
This is better, but still not quite right, missing some spacing on the left.

<img width="393" alt="Screenshot 2020-09-21 at 15 28 18" src="https://user-images.githubusercontent.com/861310/93780074-9d367900-fc1f-11ea-9677-671987c4d2b2.png">

Reply:
Fixed